pipes/reddit-auto-posts/app/api/pipeline/route.ts

```diff
@@ -0,0 +1,168 @@
+"use server";
+import fs from "node:fs";
+import { NextResponse } from "next/server";
+import { pipe } from "@screenpipe/js/node";
+import sendEmail from "@/lib/actions/send-email";
+import generateDailyLog from "@/lib/actions/generate-log";
+import generateRedditQuestions from "@/lib/actions/generate-reddit-question";
+import saveDailyLog from "@/lib/actions/savelog";
+
+export async function GET() {
+  try {
+    console.log("starting daily log pipeline");
+    const settings = await pipe.settings.getNamespaceSettings("reddit-auto-posts");
+    const interval = settings?.interval * 1000 || 60000;
+    const summaryFrequency = settings?.summaryFrequency;
+    const emailTime = settings?.emailTime;
+    const emailAddress = settings?.emailAddress;
+    const emailPassword = settings?.emailPassword;
+    const customPrompt = settings?.customPrompt!;
+    const dailylogPrompt = settings?.dailylogPrompt!;
+    const gptModel = settings?.gptModel;
+    const gptApiUrl = settings?.gptApiUrl;
+    const openaiApiKey = settings?.openaiApiKey;
```

Comment:
wrong

Reply:
https://github.com/mediar-ai/screenpipe/blob/4707469491bf5320d0384ea2259850f9fab70cef/pipes/search/src/components/search-chat.tsx#L626

also i'd prefer if you use openai sdk

Reply:
you need to support screenpipe cloud

Reply:
> you need to support screenpipe cloud

can i get the key for screenpipe cloud, just for testing purpose...